slick/src/main/scala/za/co/absa/fadb/slick/SlickQuery.scala

```diff
@@ -61,19 +60,26 @@ class SlickQueryWithStatus[R](
    */
   override def toStatusExceptionOrData(
     statusWithData: FunctionStatusWithData[PositionedResult]
-  ): Either[StatusException, R] = {
-    checkStatus(statusWithData) match {
-      case Left(statusException) => Left(statusException)
-      case Right(value)          => Right(getResult(value))
+  ): ExceptionOrStatusWithDataRow[R] = {
+    val o = checkStatus(statusWithData)
```

Comment:
Consider using more descriptive names for variables in this method.

Reply:
Oh, damned! Apologies, this is a result of me debugging this and not changing it back. Rookie mistake :D